### PR TITLE
PopupInput fixes

### PR DESCRIPTION
**Pull request by TAImatem**
Popup input seems to work correctly now (not block mouse input, come back to Mod Config Menu when closed), although opening it from pause menu is broken now smh

### CODE REVIEW

File: src/OWML.ModHelper.Menus/ModInputMenu.cs [hunk 1]
```diff
@@ -43,8 +43,9 @@ public void Open(InputType inputType, string value)
 			}
 			var message = inputType == InputType.Number ? "Write a number" : "Write some text";
 
-			_inputMenu.Invoke("InitializeMenu");
-			_inputMenu.Activate();
+			_inputMenu.EnableMenu(true);
+			//_inputMenu.Invoke("InitializeMenu");
+			//_inputMenu.Activate();
```
Comment (amazingalek) [inline src/OWML.ModHelper.Menus/ModInputMenu.cs:48]: comments? :)

File: src/OWML.ModHelper.Menus/ModInputMenu.cs [hunk 2]
```diff
@@ -60,6 +61,7 @@ public IModInputMenu Copy()
 			var newPopupObject = CopyMenu();
 			var newPopup = new ModInputMenu(Console);
 			newPopup.Initialize(newPopupObject.GetComponent<PopupInputMenu>());
+			newPopup._inputMenu.SetValue("_initialized", false);
```
Comment (amazingalek) [inline src/OWML.ModHelper.Menus/ModInputMenu.cs:64]: make a public method (and add to interface) for this?

File: src/OWML.ModHelper.Menus/ModOptionsMenu.cs [hunk 3]
```diff
@@ -85,6 +98,23 @@ private void AddSelectablePair(IModTabMenu tabMenu)
 			Menu.SetValue("_tabSelectablePairs", selectablePairs.ToArray());
 		}
 
+		private void OnTabOpen(IModTabMenu tabMenu)
+		{
+			var selectablesStack = MenuStackManager.SharedInstance.GetValue<Stack<Selectable>>("_lastSelectedStack");
+			if ((selectablesStack.Count + _menuStackCount) == MenuStackManager.SharedInstance.GetMenuCount())
```
Comment (amazingalek) [inline src/OWML.ModHelper.Menus/ModOptionsMenu.cs:104]: maybe `>=` just in case?

Comment (TAImatem) [reply src/OWML.ModHelper.Menus/ModOptionsMenu.cs:104]: Actually, it would be ≤ then

File: src/OWML.ModHelper.Menus/ModInputMenu.cs (hunk 2 above)
Comment (TAImatem) [reply src/OWML.ModHelper.Menus/ModInputMenu.cs:64]: `_initialized` is from OW. I'm mostly just replicating it calling `Initialize`(maybe I should do it in Open though). Not sure if it's even necessary TBH

Comment (amazingalek) [reply src/OWML.ModHelper.Menus/ModInputMenu.cs:62]: 🤷 👍

**Review by amazingalek — APPROVED** [review]

**Pull request merged**